sample-app/screens/HomeScreen.js

```diff
@@ -119,10 +151,11 @@ export default class HomeScreen extends React.Component<Props, State> {
 
   async makeApplePayPayment(isPayment: boolean = true) {
     const title = isPayment ? "Apple Pay payment" : "Apple Pay pre-auth";
-
+    applePayOptions.transactionType = isPayment ? RNJudo.APPLE_PAY_TRANSACTION_PAYMENT : RNJudo.APPLE_PAY_TRANSACTION_PREAUTH;
+    paymentOptions.paymentMethods = RNJudo.PAYMENT_METHOD_APPLE_GOOGLE_PAY;
```

Comment:
This shouldn't be here as we are making an Apple Pay payment here directly, not through the payment methods screen

Reply:
There is two methods, one makes a apple pay payment and other - showPayment with a button - `ApplePay`